test/y2partitioner/actions/controllers/partition_table_test.rb

```diff
@@ -34,7 +34,7 @@
 
     describe "#new" do
       it "has an initial type" do
-        expect(subject.type).to eq Y2Storage::PartitionTables::Type::MSDOS
+        expect(subject.type).to eq Y2Storage::PartitionTables::Type::GPT
```

Comment:
The output of this test is
```
describe Y2Partitioner::Actions::Controllers::PartitionTable
  With a PC with 2 disks with some partitions
    it has an initial type
```
But:
 1) The fact that is a windows PC with 2 disks is as descriptive as irrelevant from the specification point of view 
 2) More important. This is not testing that "it has an initial type". This is testing than "the initial type is GPT".

This would be the real check for the  "has an initial type" check.

```ruby
expect(subject.type).to be_a Y2Storage::PartitionTables::Type
```

Reply:
Yes, that actually makes a lot more sense; it does not depend on the default in the library.